ld: error: undefined symbol: backtrace
>>> referenced by log.c
>>>               src/libplacebo.so.192.p/log.c.o:(pl_log_stack_trace)

ld: error: undefined symbol: backtrace_symbols
>>> referenced by log.c
>>>               src/libplacebo.so.192.p/log.c.o:(pl_log_stack_trace)

--- src/meson.build.orig	2022-01-16 12:36:07 UTC
+++ src/meson.build
@@ -73,6 +73,8 @@ conf_internal.set('PL_HAVE_UNWIND', unwind.found())
 conf_internal.set('PL_HAVE_EXECINFO', cc.has_header('execinfo.h'))
 if unwind.found()
   build_deps += [unwind, cc.find_library('dl', required : false)]
+else
+  build_deps += [cc.find_library('execinfo', required : false)]
 endif
 
 # work-arounds for glslang braindeath
